src/Documenter.jl

```diff
@@ -524,6 +528,7 @@ function deploydocs(;
         devurl = "dev",
         versions = ["stable" => "v^", "v#.#", devurl => devurl],
         forcepush::Bool = false,
+        uploader = "TRAVIS"
```

Comment:
Probably better as an enum.

Reply:
Would have to export all of them then? Or you'd have to do `uploader = Documenter.APPVEYOR`?

Making it a `Symbol` would be another option.

Reply:
I think an enum might be better, since we don't want the user giving invalid input...thanks @fredrikekre!  Yeah, we'd probably want to export them - but while building docs I don't think it's going to be a problem.